mopro-ios/MoproKit/Bindings/mopro.swift

```diff
@@ -921,6 +921,15 @@ public func add(a: UInt32, b: UInt32)  -> UInt32 {
     )
```

Comment:
FYI we will probably get rid of mopro-ios and use mopro-cli - perhaps we can initialize a new project with `mopro-cli` and put in examples/templates?

Reply:
sounds great! we will start this template ASAP.

Reply:
So ideally the whole swift test app will use `mopro-cli` and in examples/templates?
@oskarth

Reply:
yes! we have the basic template for simple apps, but benchmarks should probably be separate, can init with `mopro-cli init`

Reply:
Hi @oskarth, maybe we will create one after this PR since we have to import the msm functions from this PR